code/index.html

```diff
@@ -4,19 +4,129 @@
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Link to Google Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700;900&family=Roboto:wght@100;300;400;500&display=swap" rel="stylesheet">
+
+
+    <!-- Link to FontAwesome Icons -->
+    <script src="https://kit.fontawesome.com/819208677a.js" crossorigin="anonymous"></script>
   </head>
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
+      <div class="CNN"><a href="#">CNN</a></div>
+      <div class="logo-imgs">
+        <img id="logo1" class="logo logo1" src="virus.png">
+        <img class="logo logo2" src="virus2.png">
+      </div>
+      <div class="soc-med">
+        <a href="https://twitter.com/cnn">
+          <i class="fab fa-twitter fa-2x soc-icon"></i>
+        </a>
+        <a href="https://www.instagram.com/cnn/?hl=en">
+          <i class="fab fa-instagram fa-2x soc-icon"></i>
+        </a>
+        <a href="https://edition.cnn.com/feedback">
+          <i class="fas fa-envelope fa-2x soc-icon"></i>
+        </a>
+        <a href="https://www.facebook.com/cnn/">
+          <i class="fab fa-facebook-f fa-2x soc-icon"></i>
+        </a>
+      </div>
     </header>
-
+    <div class="black-stripe"></div>
+    <nav class="navbar">
+        <a class ="nav-links" href="#">World</a>
+        <a class ="nav-links" href="#">Europe</a>
+        <a class ="nav-links" href="#">Culture</a>
+        <a class ="nav-links" href="#">Sports</a>
+        <a class ="nav-links" href="#">Technology</a>
+    </nav>
+      
+    <h1>COVID NEWS NOW</h1>
+    
+    <!-- The big news content -->
     <section class="big-news">
-      <!-- The big news content -->
+      <div class="big-news-pic"><img class="big-pic" src="https://picsum.photos/600">
+      </div>
+      <div class="big-news-text"><h2 class="biggest-heading">EU SET TO FOLLOW SWEDEN'S COVID STRATEGY: EUROPEANS RISING UP AGAINST LATEST SHUT DOWNS </h2></div>
     </section>
-
-    <section class="">
-      <!-- The other news  -->
+    
+    <section class="cards">
```

Comment:
Also here if you want to you can add more covid-related images, and change the lorem ipsum text to more article-ish text. You could also make the card into a link so it looks clickable...  or maybe the "more"-button instead, and I noticed on CNNs site that they have red hover-effect on the text if you want it to look similar.